### ISSUE TITLE
About training and benchmark on new flow dataset

### ISSUE THREAD

**Issue by routineLife1** [XiaoyuShi97/VideoFlow#19]
Nice work!, VideoFlow performs very well in lots of 3d animation. Btw, the [Spring optical flow](https://spring-benchmark.org/) dataset was recently released at CVPR2023. Do you have any plans to finetune or train a new model on Spring?

**Comment by XiaoyuShi97** [comment]
Hi. Months ago, I tested videoflow on spring training set without fine-tuning and the epe is sub-pixel (~0.6?). I guess finetuning on spring may not bring a huge performance difference. Do you have other concerns or reasons?

**Comment by routineLife1** [comment]
Okay, I have seen that this is indeed the case in the past few days, The general motion amplitude of the Spring dataset may be relatively small.And, how about training a model on AnimeRun？This is very likely to improve the accuracy of 2D animation optical flow prediction.
